app/build.gradle

```diff
@@ -1,5 +1,11 @@
 apply plugin: 'com.android.application'
 
+apply from: "$project.rootDir/tools/git-hooks.gradle"
+apply from: "$project.rootDir/tools/checkstyle.gradle"
+apply from: "$project.rootDir/tools/findbugs.gradle"
+apply from: "$project.rootDir/tools/pmd.gradle"
+apply from: "$project.rootDir/tools/lint.gradle"
+
```

Comment:
Include the dependencies of the `check` command.
`check.dependsOn 'checkstyle', 'findbugs', 'pmd', 'lint'`